field_manager/field_manager.F90

```diff
@@ -224,7 +225,8 @@ module field_manager_mod
 public :: fm_get_value         !< (entry, value [, index]) return success !! generic
 public :: fm_get_value_integer !<   as above (overloaded function)
 public :: fm_get_value_logical !<   as above (overloaded function)
-public :: fm_get_value_real    !<   as above (overloaded function)
+public :: fm_get_value_real_r4 !<   as above (overloaded function)
+public :: fm_get_value_real_r8 !<   as above (overloaded function)
```

Comment:
Same comment `fm_get_value` should be the only public function

Reply:
Agree but I'm scared to change it.  Coupler_mod in FMS calls fm_util_get_real instead of "fm_util_get_value" so for consistency, I think I'm going to leave this in here.  But maybe this can be a separate issue, cleaning up interfaces in field_manager?

Reply:
yeah, this should be a separate issue, not related to mixed mode.